vms/shift/templates/shift/cancel_email.txt

```diff
@@ -0,0 +1,3 @@
+{% autoescape off %}
+{{admin_first_name}} {{admin_last_name}} has cancelled your shift starting from {{shift_start_time}} to {{shift_end_time}} on {{shift_date}} of {{job_name}} during the {{event_name}} event. For any questions or concerns, email {{admin_email}}.
```

Comment:
{{admin_first_name}} {{admin_last_name}} has cancelled your shift scheduled for {{shift_date}} from {{shift_start_time}} to {{shift_end_time}} of {{job_name}} for the event {{event_name}}. For any questions or concerns, email {{admin_email}}.
I think you can change the message to above sentence.